ppd/ppd-ipp.c

```diff
@@ -657,15 +657,25 @@ ppdLoadAttributes(
 	  yres = res_y[0];
 	}
       }
-      else if (strlen(items[0]) > 0 &&
-	       ((int)(strtol(items[0], &p, 10) * 0) ||
-		(errno == 0 && *p == '\0')))
+      else if (items[0][0] && isdigit((int)items[0][0]))
```

Comment:
This means that you consider an entry as a series of integers just when the first character of the first item is a digit?

And also why the cast to integer in `isdigit((int)items[0][0])`?

Reply:
Ad 1 - yes - in case there are some items like "1wide", strtol catches them in 'p' - I agree it expects the values in the attributes we are interested (urf-supported, pwg, pclm) in in this scope to don't look like this, because such values will be thrown away, I'll rewrite it to take this into account, so I'll use strtol() again.

However I will try at least to save the first value and use bitwise OR in conditional (+explanation why) - the bitwise operation should be more efficient.